pages/guides/docker_containerized_builds.md.erb

```diff
@@ -45,12 +45,12 @@ app:
     - ./:/go/src/github.com/buildkite/agent
     - /usr/bin/buildkite-agent:/usr/bin/buildkite-agent
   environment:
-    - BUILDKITE_AGENT_ACCESS_TOKEN
-    - BUILDKITE_JOB_ID
-    - BUILDKITE_BUILD_ID
-    - BUILDKITE_BUILD_NUMBER
-    - GITHUB_RELEASE_ACCESS_TOKEN
-    - DOCKER_HUB_TRIGGER_TOKEN
+    - BUILDKITE_AGENT_ACCESS_TOKEN: $BUILDKITE_AGENT_ACCESS_TOKEN
```

Comment:
Pretty sure they're still supported… I couldn't see any word on the Github issues about it going away https://docs.docker.com/compose/compose-file/#environment